src/libtexture/texture_pvt.h

```diff
@@ -374,16 +374,12 @@ class TextureSystemImpl : public TextureSystem {
         return texturefile;
     }
 
-    /// Find the tile specified by id.  If found, return true and place
-    /// the tile ref in thread_info->tile; if not found, return false.
-    /// This is more efficient than find_tile_main_cache() because it
-    /// avoids looking to the big cache (and locking) most of the time
-    /// for fairly coherent tile access patterns, by using the
-    /// per-thread microcache to boost our hit rate over the big cache.
-    /// Inlined for speed.
-    bool find_tile(const TileID& id, PerThreadInfo* thread_info)
+    /// Find the tile specified by id.  Just a pass-through to the
+    /// underlying ImageCache.
+    bool find_tile(const TileID& id, PerThreadInfo* thread_info,
+                   bool mark_same_tile_used = true)
```

Comment:
Change this one too?

Reply:
yes, and oops, I found a place where I didn't do the optimization. Results may be even better, will have to run the benchmarks again!

Reply:
Results were virtually the same. The vast majority of the contention was apparently covered by the first set of changes.